WorkcraftCore/src/org/workcraft/util/FileUtils.java

```diff
@@ -70,15 +70,10 @@ public static void copyFileToStream(File in, OutputStream out) throws IOExceptio
         FileInputStream is = new FileInputStream(in);
         FileChannel inChannel = is.getChannel();
         WritableByteChannel outChannel = Channels.newChannel(out);
-        try {
-            inChannel.transferTo(0, inChannel.size(), outChannel);
-        } catch (IOException e) {
-            throw e;
-        } finally {
-            if (is != null) is.close();
-            if (inChannel != null) inChannel.close();
-            if (outChannel != null) outChannel.close();
-        }
+        inChannel.transferTo(0, inChannel.size(), outChannel);
+        if (is != null) is.close();
```

Comment:
No, the PMD check is good. Just wanted to point out that the `finally` section is needed (not the `catch` though). A better code would be:
```
    public static void copyFileToStream(File in, OutputStream out) throws IOException {
        FileInputStream is = new FileInputStream(in);
        FileChannel inChannel = is.getChannel();
        WritableByteChannel outChannel = Channels.newChannel(out);
        try {
            inChannel.transferTo(0, inChannel.size(), outChannel);
        } finally {
            if (is != null) is.close();
            if (inChannel != null) inChannel.close();
            if (outChannel != null) outChannel.close();
        }
    }
```